qiskit/providers/models/pulsedefaults.py

```diff
@@ -144,7 +144,15 @@ def __init__(self,
         self.converter = QobjToInstructionConverter(pulse_library)
         for inst in cmd_def:
             pulse_insts = [self.converter(inst) for inst in inst.sequence]
-            schedule = ParameterizedSchedule(*pulse_insts, name=inst.name)
+            # there is mismatch among instruction sequence and parameter ordering
+            # in u2 and u3 gates.
+            parameters = []
+            if inst.name in ['u2', 'u3']:
```

Comment:
I don't like seeing this -- let's try to fix the root cause

Reply:
I think this is temporary fix. Adding `parameters` field in `Command` Qobj is the best solution, ie `ParametrizedSchedule(*pulse_insts, name=inst.name, parameters=inst.parameters)`, but this needs spec update.

Reply:
I agree that a signature should have been defined for the cmd_def.